package.json

```diff
@@ -7,11 +7,12 @@
   "scripts": {
     "commit": "git-cz",
     "prepare": "grunt release",
+    "lint": "grunt lint && prettier --check 'src/**/*.ts'",
+    "lint:fix": "prettier --write 'src/**/*.ts'",
     "test": "nyc mocha",
```

Comment:
It looks like some tasks are done via `grunt`, some are called directly via scripts. If general direction is to do everything via scripts, then I could remove `eslint` task from `Gruntfile.js` completely and call linters from scripts too.